py/jupyterlite/src/jupyterlite/manager.py

```diff
@@ -0,0 +1,191 @@
+"""Manager for JupyterLite
+"""
+
+import doit
```

Comment:
Should this be added to the install requires in `pyproject.toml`?

Reply:
haha! a527085